content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,300 @@
+
+Once the Agent is installed, it's graphically represented within the [Infrastructure List][23] with a bone icon:
+{{< img src="getting_started/integrations/infrastructure-list.png" alt="An image from the infrastructure list of the Datadog account. Two hosts are displayed in a list format. Both hosts show the AWS icon for the AWS integration and 'aws' shown in a blue box to show they are associated with the AWS integration. One host also shows a dog-bone icon and blue boxes for 'ntp' and 'system'.">}}
+
+As can be seen in the screenshot above, the host with the Datadog Agent is reporting data from the [System][24] and [NTP][25] checks. The System check provides metrics around CPU, memory, filesystem, and I/O, providing additional insights into the host. You can enable additional [integrations][26] to suit the environment and use case, and [DogStatsD][27] can additionally be leveraged for sending custom metrics directly to Datadog. 
+
+See the [FAQ on why you should install the Datadog Agent on your cloud instances][28] for more information about the benefits of this approach.
+
+### Using the Datadog Agent with Amazon Container Services
+
+For containerized environments, you can use the Datadog Agent whether you're managing your instances or leveraging [Fargate][29] for a serverless environment.
+
+#### ECS with EC2 launch type
+
+Use the [Amazon ECS documentation][30]  to run the [Datadog Docker Agent][31] on the EC2 instances in your ECS cluster. Review the [Amazon ECS Data Collection documentation][32]  to see the metrics and events reported to your Datadog account.
+
+#### ECS with Fargate launch type
+
+Use the [Amazon ECS on AWS Fargate documentation][33]  to run the Agent as a container in the same task definition as your application. **Note**: Datadog Agent version 6.1.1 or higher is needed to take full advantage of the Fargate integration.
+
+#### EKS
+
+You don't need any specific configuration for AWS Elastic Kubernetes Service (EKS), as mentioned in the [Kubernetes Distributions documentation][34]. Use the [dedicated Kubernetes documentation][35] to deploy the agent in your EKS cluster.
+
+#### EKS with Fargate
+
+Since Fargate pods are managed by AWS, they exclude host-based system checks like CPU and memory. In order to collect data from your AWS Fargate pods, use the [Amazon EKS on AWS Fargate documentation][53] to run the Agent as a sidecar of your application pod with custom role-based access control (RBAC). **Note**: This requires Datadog Agent version 7.17 or higher.
+
+#### EKS Anywhere
+
+Use the [EKS Anywhere documentation][36] for on-premises Kubernetes clusters.
+
+## What's next?
+
+### Create additional Datadog resources
```

Comment:
I would actually reorganize "What's next" a bit where we do have a "Get more from the Datadog plaform" which includes the Datadog agent and other compute monitoring as well as "Create additional Datadog resources" section talking about CloudFormation Registry, setting up monitors and dashboard etc. looking at the out of the box dashboards we have for other AWS servuces

and then a separate "Explore Related products" section with APM, Cloud SIEM, and Cloud Security Posture Management. And then also move Serverless to there.